test/basicQuestions.js

```diff
@@ -70,6 +70,15 @@ describe('basicQuestions', function() {
         });
     });
 
+    it('returns correct answer for what is xyz\' population question', function(done) {
+
+        app.answer('What is the Germany\'s population? ', function(answer) {
+            assert.equal(answer.interpretation, 'population of Germany?');
+            assert.equal(answer.result, '81292400');
```

Comment:
this will fail when too many people will die, be born or move, right? (assuming somebody is keeping wikidata roughly in sync witht hat)